tests/readme.rs

```diff
@@ -0,0 +1,124 @@
+mod fixtures;
+
+use assert_cmd::prelude::*;
+use assert_fs::fixture::{FileWriteStr, TempDir};
+use assert_fs::prelude::PathChild;
```

Comment:
Can you merge this to:
```suggestion
use assert_fs::prelude::*;
```
?
I'm not sure the `prelude` has all the required packages but that's usually what I try first.

Reply:
```
error[E0412]: cannot find type `TempDir` in this scope
  --> tests/readme.rs:28:33
   |
28 | fn show_readme_contents(tmpdir: TempDir) -> Result<(), Error> {
   |                                 ^^^^^^^ not found in this scope
   |
help: consider importing this struct
   |
3  | use assert_fs::TempDir;
   |

error[E0412]: cannot find type `TempDir` in this scope
  --> tests/readme.rs:71:45
   |
71 | fn show_readme_contents_directories(tmpdir: TempDir) -> Result<(), Error> {
   |                                             ^^^^^^^ not found in this scope
   |
help: consider importing this struct
   |
3  | use assert_fs::TempDir;
```
just having `use assert_fs::prelude::*;` instead of those two gives me this error.

Reply:
Ah yes, nevermind then.